firefox-ios/Client/Frontend/Home/Homepage Rebuild/SectionHeader/SectionHeaderState.swift

```diff
@@ -0,0 +1,14 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import Redux
+
+struct SectionHeaderState: Equatable, Hashable {
```

Comment:
in regards to naming these `state`, will create a separate PR depending on what the team agrees upon